src/style.css

```
@@ -0,0 +1,29 @@
+body {
+  font-size: 13px;
+  font-family: Arial;
+}
+
+p {
+  margin: 0;
+}
+
+ol {
+  margin: 0;
+}
```

you can combine thos styles
```suggestion
p, ol {
  margin: 0;
}

```